### PR TITLE
Testplan related bugfixes

### PR DESCRIPTION
**Pull request by ldoktor**
Hello guys,

there are two little issues found while running the manual testplan.

Regards,
Lukáš

### CODE REVIEW

File: avocado/core/html.py [hunk 1]
```diff
@@ -130,19 +130,20 @@ def tests(self):
                    "INTERRUPTED": "danger"}
         test_info = []
         results_dir = self.results_dir(False)
-        for t in self.result.tests:
-            formatted = {}
-            logdir = os.path.join(results_dir, 'test-results', t['logdir'])
-            formatted['logdir'] = os.path.relpath(logdir, self.html_output_dir)
+        for tst in self.result.tests:
+            tst = tst.copy()    # we don't want to override other's results
```
Comment (clebergnu) [inline avocado/core/html.py]: Previous comment:

--

This change the `result.tests` entries, which then break expecations of other parts of the code.  One example is:

```
  File "/home/travis/build/avocado-framework/avocado/avocado/core/html.py", line 142, in tests

    time.localtime(tst['time_start']))

TypeError: a float is required
```

That's why I created the `formatted` dict.  But yeah, it misses some of the items, and the failure is a silent field on the generated HTML.

The way to go here is to keep `formatted` and add the missing fields.

--

But yeah, copying is not that bad.

Comment (ldoktor) [inline avocado/core/html.py]: I wanted to use `copy.deepcopy`, but when I looked at it it looked safe to use just a plain `copy`. Did you found those issues, should I change it to the `deepcopy` again?

File: avocado/utils/filelock.py [hunk 2]
```diff
@@ -35,7 +35,7 @@ class FileLock(object):
 
     """
     Creates an exclusive advisory lock for a file.
-    All processes should would use and honour the advisory
+    All processes should would use and honor the advisory
```
Comment (adereis) [inline avocado/utils/filelock.py]: You're changing the spelling, but there's actually a grammar mistake in this sentence ("should would") which you're missing.

Honour and honor are both valid. The former is more common in en_GB, while the latter is en_US.

File: avocado/core/html.py (hunk 1 above)
Comment (clebergnu) [inline avocado/core/html.py]: I was referring to the older version (without `copy`).

File: avocado/utils/filelock.py (hunk 2 above)
Comment (ldoktor) [inline avocado/utils/filelock.py]: Oh, good catch

**Pull request closed without merging**